plugin/pkg/scheduler/scheduler.go

```diff
@@ -92,6 +97,12 @@ func (s *Scheduler) scheduleOne() {
 		glog.V(1).Infof("Failed to schedule: %+v", pod)
 		s.config.Error(pod, err)
 		s.config.Recorder.Eventf(pod, api.EventTypeWarning, "FailedScheduling", "%v", err)
+		s.config.PodConditionUpdater.Update(pod, &api.PodCondition{
```

Comment:
I wonder if we could make this async? Even though this is error path, having it block/delay scheduling other pods is bad.